downstream/modules/controller/proc-controller-delete-user.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+[id="proc-delete-user"]
+
+= Deleting a user from {ControllerName}
+
+When you delete a user account, the name and email of the user are permanently removed from {ControllerName}.
```

Comment:
Suggestion: Replace the word `are` with `is`.  
Example: `are permanently removed` -> `is permanently removed`